nettacker/modules/scan/adobe_aem_lastpatcheddate.yaml

```
@@ -0,0 +1,48 @@
+                - 80
+                - 443
+        response:
+          condition_type: and
+          log: "response_dependent['headers']['Last-Modified']"
+          conditions:
+            status_code:
+              regex: "200"
+              reverse: false
+            headers:
+              Last-Modified:
+                regex: .*  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)